adapters/liftoff/liftoff.go

```diff
@@ -0,0 +1,180 @@
+package liftoff
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const SupportedCurrency = "USD"
+
+type adapter struct {
+	Endpoint string
+}
+
+type prebid struct {
+	Debug bool `json:"debug"`
+}
+
+type prebidExt struct {
+	Prebid prebid `json:"prebid"`
+}
+
+type liftoffImpressionExt struct {
+	*adapters.ExtImpBidder
+	// Ext represents the vungle extension.
```

Comment:
What's vungle?

Reply:
`Vungle` is our company name before merging with liftoff. In our server code it's a field name in `imp.ext.vungle` which holds our external partners' token and other info.